### PR TITLE
Support test execution on RHEL9

### PR DESCRIPTION
**Pull request by alishaIBM**

### CODE REVIEW

File: tools/common.sh [hunk 1]
```diff
@@ -49,6 +49,7 @@ function os_type()
         grep -i -q "Red Hat .* 6" "${fileName}" && { echo "rhel6"; return 0; }
         grep -i -q "Red Hat .* 7" "${fileName}" && { echo "rhel7"; return 0; }
         grep -i -q "Red Hat .* 8" "${fileName}" && { echo "rhel8"; return 0; }
+	grep -i -q "Red Hat .* 9" "${fileName}" && { echo "rhel9"; return 0; }
```
Comment (liangxia) [inline tools/common.sh:52]: Could you fix the idention？

File: tools/install_os_deps.sh [hunk 2]
```diff
@@ -28,6 +28,10 @@ elif [ "$(os_type)" == "rhel8" ] || [ "$(os_type)" == "centos8" ]; then
     cmd="dnf install -y --setopt=skip_missing_names_on_install=False --allowerasing --skip-broken"
     file="deps.yum.RHEL8"
     additional_deps=install_rvm_if_ruby_is_outdated
+elif [ "$(os_type)" == "rhel9" ] || [ "$(os_type)" == "centos9" ]; then
+    cmd="dnf install -y"
+    file="deps.yum.RHEL9"
```
Comment (liangxia) [inline tools/install_os_deps.sh:33]: Content of the file ?

File: tools/common.sh (hunk 1 above)
Comment (alishaIBM) [reply tools/common.sh]: Acknowledge

**Pull request merged**